expose error object to developers and add docs (#15)
podium-lib/labs: packages/fastify-podlet-server/README.md

```

See the [Fastify docs](https://www.fastify.io/docs/latest/Reference/Validation-and-Serialization/) for more

> ## Errors

Reviewer: ```suggestion
## Error handling
```

> Error handler has been added to enable developers to control application errors.

Reviewer 2: Is this rather intended to start with `An error handler` or `Error handling`?

Author: Whoops 👍

> Error handler has been added to enable developers to control application errors.

Author: ```suggestion
Error handling has been added to enable developers to control application errors.
```

> If you throw an error from app.setContentState or app.setFallbackState this will result in a 500 error. 

Reviewer: ```suggestion
If you throw an ordinary error from app.setContentState or app.setFallbackState this will result in the route serving a http 500 Internal Server error. 
```

> If you throw an error with the appropriate http status, this will result in a matching error.

Reviewer: ```suggestion
You can control which http errors the server should respond with by throwing a [http-errors](https://www.npmjs.com/package/http-errors) error. 
```

> In addition, an errors object with convenience methods has been added to make throwing different kinds of http errors more streamlined

Reviewer: ```suggestion
In addition, an errors object with http-errors convenience methods has been added to make throwing different kinds of http errors more streamlined
```
